resources/warnings.json

```
@@ -5469,7 +5469,7 @@
     "versions": [
       {
         "lastVersion": "2.12.0",
-        "pattern": ".*"
+        "pattern": "(2[.]1[0-2]|2[.][0-9])([.-].+|)"
```

```suggestion
        "pattern": "(1|2[.]1[0-2]|2[.][0-9])([.-].+|)"
```

Version 1.0 existed but is not currently published, better safe than sorry.